modules/web/src/app/dynamic/enterprise/quotas/quota-widget/component.ts

```diff
@@ -139,6 +139,12 @@ export class QuotaWidgetComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   updateEstimatedQuota(quota: ResourceQuotaCalculation): void {
+    if (!quota) {
+      this.isEstimatedQuotaExceeded = false;
+      this.estimatedQuotaExceeded.emit(false);
```

Comment:
We should also clear `this.estimatedQuota` and `estimatedQuotaPercentage` values.